pkg/syncer/spec/spec_process.go

```diff
@@ -222,6 +222,9 @@ func (c *Controller) ensureSyncerFinalizer(ctx context.Context, gvr schema.Group
 }
 
 func (c *Controller) applyToDownstream(ctx context.Context, gvr schema.GroupVersionResource, downstreamNamespace string, upstreamObj *unstructured.Unstructured) error {
+	if gvr.Group == "" && gvr.Version == "v1" && gvr.Resource == "namespaces" {
```

Comment:
?

Reply:
@ncdc suggested me to add that line as there is another bug somewhere in the code that needs to be fixed.